rasa/cli/data.py

```diff
@@ -643,15 +643,23 @@ def _backup(path: Path) -> None:
 
 
 def _print_success_message(new_rules: List["StoryStep"], output_file: Path) -> None:
-    if len(new_rules) > 1:
-        suffix = "rule"
-        verb = "was"
-    else:
+    if len(new_rules) > 1 or len(new_rules) == 0:
         suffix = "rules"
         verb = "were"
+    else:
+        suffix = "rule"
+        verb = "was"
 
     rasa.shared.utils.cli.print_success(
-        f"Finished migrating your policy configuration 🎉.\n"
-        f"The migration generated {len(new_rules)} {suffix} which {verb} added to "
-        f"'{output_file}'."
+        "Finished migrating your policy configuration 🎉."
     )
+    if len(new_rules) == 0:
```

Comment:
Can we just have one set of `if` and `else`/`elif`s

Reply:
Not really because we have different criteria in each check. More specifically : 
- the variables 'rules_text' and 'verb' are changing if the rules are >1 or equal to 0 **vs** we have only 1 rule (else case). 
- but then the actual success message is different only in the case that we have 0 rules (_...so **no** rules are added...._) **otherwise** the syntax/text is the same (case of having >1 rules or only 1 rule).